im-crawler/src/main/java/com/wipro/ats/bdre/imcrawler/frontier/InProcessPagesDB.java

```diff
@@ -48,8 +46,7 @@
     public InProcessPagesDB() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)